frontend/components/AddHostsModal/PlatformWrapper/PlatformWrapper.tsx

```diff
@@ -338,6 +338,14 @@ const PlatformWrapper = ({
   }
 }`,
     };
+    const getHelpTextForPackageType = (): string => {
+      if (packageType === "deb") {
+        return " For CentOS, Red Hat, and Fedora Linux, use --type=rpm.";
+      } else if (packageType === "msi") {
+        return " Windows can only generate an MSI package.";
```

Comment:
Not sure that this conveys information correctly, since we're talking about _building_ an MSI package, which any OS can do (with the correct dependencies), not building _from_ a specific OS.

Reply:
@iansltx I had attached a wrong issue,the correct issue number is  #22022

Reply:
Yeah, I'm specifically commenting here about the copy for this line; swapping "enroll the host" to "build the installer package" makes sense. I made some updates to the guide linked here recently, hence being picky about how we word things.

Reply:
@iansltx ,I think I missed the link would you mind sharing the link again?Also,I have updated the PR